webapp/client/src/stories/FormFieldCheckBox.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import FormFieldCheckBox from "../components/FormFieldCheckBox";
+import StepForm from "../components/StepForm";
+import { Default as StepFormDefault } from "./StepForm.stories";
+
+export default {
+  title: "Form Fields/CheckBox",
+  component: FormFieldCheckBox,
+};
+
+const Template = (args) => (
+  <StepForm {...StepFormDefault.args}>
+    <FormFieldCheckBox {...args} />
+  </StepForm>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  fieldId: "checkBox",
+  fieldName: "checkBox",
+  labelText: "Merkzeichen G",
+  errors: [],
+  values: [],
```

Comment:
values? Copy pasta? I think we want `checked`no?

Reply:
I removed it